File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/event/UserResourceRemovedEvent.java
package work.socialhub.kslack.entity.event;

/**
 * We send this event when a user removes an existing grant for your workspace app.
 * <p>
 * https://api.slack.com/events/user_resource_removed
 */
public class UserResourceRemovedEvent implements Event {

    public static final String TYPE_NAME = "user_resource_removed";

    private final String type = TYPE_NAME;
    private String user;
    private String triggerId;

    public String getType() {
        return this.type;
    }

    public String getUser() {
        return this.user;
    }

    public String getTriggerId() {
        return this.triggerId;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public void setTriggerId(String triggerId) {
        this.triggerId = triggerId;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/event/UserResourceRemovedEvent.kt
package work.socialhub.kslack.entity.event

/**
 * We send this event when a user removes an existing grant for your workspace app.
 *
 *
 * https://api.slack.com/events/user_resource_removed
 */
class UserResourceRemovedEvent : Event {
    override val type: String = TYPE_NAME
    var user: String? = null
    var triggerId: String? = null

    companion object {
        const val TYPE_NAME: String = "user_resource_removed"
    }
}